javascript/big-practice/src/scripts/views/listNoteView.js

```diff
@@ -460,15 +444,15 @@ export default class ListNoteView {
       if (!selectedElement) {
         e.target.parentElement.classList.add('selected');
         this.elementHelpers.countAndShowSelected(countNotesSelected);
-        this.headerAfterSelect.style.transform = 'translateY(-100%)';
+        this.elementHelpers.translateYElement(this.headerAfterSelect, '-100');
       } else {
         e.target.parentElement.classList.remove('selected');
```

Comment:
I see this action is the same as the show/hide element. In general, we are handling add/remove classes for an element. So you can define a common method for add and one for remove.